src/error.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+$config = include 'config.php';
+
+if (isset($_GET['error_msg'])) {
```

Comment:
Minor thing but this line can just be 
`$error_msg = $_GET['error_msg'] ?? null;`

Reply:
I'm not sure if it can be in this case since I only want to set `error_msg` if the query parameter is set, otherwise I expect the `error_msg` to be set by the code that included the file.
For example, the `show_error_page` function:
```php
function show_error_page($message) {
    $error_msg = $message;
    include 'error.php';
    die();
}
```

Reply:
However, it can be 
```php
$error_msg = $_GET['error_msg'] ?? $error_msg;
```
I've committed that change.